tests/certification/bindings/zeebe/command/components/standard/zeebe-command.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: zeebe-command
+spec:
+  type: bindings.zeebe.command
```

Comment:
Should this file be used as a conformance test or can a conformance test be added as well? Ideally, for the components we will want both explicit `certification` and `conformance` tests. Following the [doc here](https://github.com/dapr/components-contrib/blob/master/tests/conformance/README.md), conformance tests would entail a `tests/config/bindings/zeebe/bindings.yml`.

Reply:
Components of type bindings contain a  bit of everything, so conformance tests are often not much useful. If we have a good, comprehensive cert test, that's _usually_ enough for a binding, especially for something that's more "unique" like Zeebe

Reply:
Oh, I see, so even though our [certification lifecycle doc](https://docs.dapr.io/operations/components/certification-lifecycle/) specified that for `beta` stage we have `conformance tests` and for `alpha` we have `certification tests` having it as is, and no _explicit_ conformance tests, is sufficient for the `beta` stage?

Reply:
That document is correct, but we need to consider the specifics of each situation. A conformance test exists to ensure that all components behave similarly and can be used "interchangeably". This is very useful for things like state stores, secret stores, etc.

However, this binding has about a dozen operations that are unique to this component, so there's not much to "conform to".

Eventually, the decision to make this beta or stable will be at the maintainers' discretion.

Reply:
Great, thanks for explaining. I'm looking into the issues related to moving the zeebe command binding & zeebe jobworker binding to `stable` is why I was asking.

Reply:
@cicoyle Please, add this binding to conformance test to keep us following the process. I understand that we had discussions about how much value conformance tests can actually add to bindings, but I would prefer to keep the process until we reach a formal decision. A change to the certification process should go into proposals IMO since it is big enough.

Reply:
@artursouza adding a conformance test doesn't add value here since there isn't any standardized behavior and interface being tested in bindings. It just means taking up more GitHub actions runners.

I think this PR here is good enough to make the component stable.